src/components/networked.js

```diff
@@ -427,6 +428,7 @@ AFRAME.registerComponent('networked', {
     syncData.template = data.template;
     syncData.persistent = data.persistent;
     syncData.parent = this.getParentId();
+    syncData.attachToParentId = data.attachToParentId;
```

Comment:
I see we already have `syncData.parent` here, so adding a new field to sync a parent id seems redundant. I think it would be better to modify `getParentId` and `initNetworkParent` here, and keep the `addEntityToPage(entity, parentId)` signature.

Reply:
If we wanted to do that I think we would need to add smt like a prefix for non-networked parents in `getParentId()` like `nonNetworked:${ParentId}` 

since over [here](https://github.com/networked-aframe/networked-aframe/blob/f1c62b41a4a13c6d4d6f425eaf14c81ab90e44ff/src/NetworkEntities.js#L93): where we do `    var parentNotCreatedYet = parent && !this.hasEntity(parent);`, . 

there we would have to parse to check if the parentId on `syncData` is a networked parent or a non-networked parent. (We have to do this because we have two cases, 1) The networked parent is not in the scene yet and we add to childCache, or 2) the parent is not a networked component so we can just add it to the scene right now)